ml/Dockerfile.gpu

```
@@ -0,0 +1,24 @@
+# GPU worker — one-shot batch task for ECS RunTask
+FROM nvidia/cuda:12.2-runtime-ubuntu22.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)